WordPress/Classes/ViewRelated/NUX/SignupEpilogueViewController.swift

```diff
@@ -73,7 +73,18 @@ class SignupEpilogueViewController: NUXViewController {
     // MARK: - analytics
 
     private func tracksProperties() -> [AnyHashable: Any] {
-        let source = socialService != nil ? "google" : "email"
+        let source: String = {
+            guard let service = socialService else {
+                return "email"
+            }
+            switch service {
```

Comment:
You could just return `service.rawValue` here

Reply:
Actually, `SocialService` does not have a `rawValue` (you may be thinking of `SocialServiceName`, which does).

Reply:
D'oh, thanks :)